apps/dsp-app/src/app/workspace/search/advanced-search/resource-and-property-selection/search-select-property/specify-property-value/specify-property-value.component.spec.ts

```diff
@@ -323,45 +323,45 @@ describe('SpecifyPropertyValueComponent', () => {
         ).toEqual(8);
     });
 
-    // it('should set the correct comparison operators for a linking property type (on top level)', () => {
```

Comment:
Is this dead code now? I'd delete it if it's not used.

Reply:
No, I uncommented it so the test is valid for t